media/apps.html

```diff
@@ -2,23 +2,74 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Media application development</title>
+    <title>Device interoperability</title>
   </head>
   <body>
     <header>
-      <h1>Media application development</h1>
+      <h1>Device interoperability</h1>
       <p>
-        The Web platform is under constant evolution. Device implementations and application developers need to share a common understanding of the functionalities that the platform offers at any given point in time, and of recommended guidance to use these powerful features.
+        Televisions, set-top boxes and other media devices typically ship with hardware architectures that are both extremely powerful to decode and render audio and video, and fairly constrained when it comes to running &mdash;and updating&mdash; applications. Web browser code bases need to be ported to these devices, and integrators need to answer a number of practical questions to adapt the code to practical device constraints. That process also usually happens early in the production cycle. As such, final products may run a code base release that is a few years old with varying degrees of APIs support and quality of implementations. In constrast, personal computers and mobile devices are tailored to run applications and follow the <a href="https://www.w3.org/2001/tag/doc/evergreen-web/">constant evolution</a> of the Web platform, with always up-to-date Web browsers. This creates interoperability issues within media devices as well as between media devices and other types of devices.
```

Comment:
I'd geneally write 'code base' as 'codebase'.

Reply:
I used "code base" because that's how it gets spelled in the [Web Media API](https://w3c.github.io/webmediaapi/) specification. But I don't disagree and would instinctively write "codebase" as well. I'll make the change.